pyanaconda/ntp.py

```diff
@@ -41,27 +45,68 @@
 #treat pools as four servers with the same name
 SERVERS_PER_POOL = 4
 
+# Description of an NTP server status.
+NTP_SERVER_STATUS_DESCRIPTIONS = {
+    NTP_SERVER_OK: N_("status: working"),
+    NTP_SERVER_NOK: N_("status: not working"),
+    NTP_SERVER_QUERY: N_("checking status")
+}
+
+log = get_module_logger(__name__)
+
 
 class NTPconfigError(Exception):
     """Exception class for NTP related problems"""
     pass
 
 
-def ntp_server_working(server):
+def get_ntp_server_summary(server, states):
+    """Generate a summary of an NTP server and its status.
+
+    :param server: an NTP server
+    :type server: an instance of TimeSourceData
+    :param states: a cache of NTP server states
+    :type states: an instance of NTPServerStatusCache
+    :return: a string with a summary
     """
-    Tries to do an NTP request to the $server (timeout may take some time).
+    return "{} ({})".format(
+        server.hostname,
+        states.get_status_description(server)
+    )
 
-    :param server: hostname or IP address of an NTP server
-    :type server: string
-    :return: True if the given server is reachable and working, False otherwise
-    :rtype: bool
 
+def get_ntp_servers_summary(servers, states):
```

Comment:
Unless I'm mistaken, this is not used anywhere?

Reply:
It is used in TUI.